### PR TITLE
chore(user): generate personal organization for new users

### PR DESCRIPTION
**Pull request by xjacka**
https://github.com/i-am-bee/internal/issues/20

Needs to add `IBM_ORGANIZATION_OWNER_ID` (or rename `ORGANIZATION_OWNER_DEFAULT_ID`)

Needs UI changes.

### CODE REVIEW

**Review by pilartomas — APPROVED** [review]

File: src/users/users.service.ts [hunk 1]
```diff
@@ -52,36 +56,73 @@ export async function createUser({
...
-    );
-  const defaultProjectAdmin = await ORM.em
-    .getRepository(ProjectPrincipal)
-    .findOneOrFail(
-      { id: PROJECT_ADMIN_ID_DEFAULT },
-      { filters: { projectAdministrationAccess: false } }
-    );
+  let orgUser: OrganizationUser | null = null;
+  let organization: Organization | null = null;
+  if (email && email.endsWith('@ibm.com')) {
```
Comment (matoushavlena) [inline src/users/users.service.ts:72]: This is not enough. Valid IBM emails are also:
`partner.ibm.com`, `us.ibm.com`, `cz.ibm.com`, etc.

**Review by matoushavlena — CHANGES_REQUESTED** [review]

**Review by pilartomas — CHANGES_REQUESTED** [review]
All looks good except the email condition 👍

**Review by pilartomas — APPROVED** [review]

File: src/users/users.service.ts (hunk 1 above)
Comment (xjacka) [reply src/users/users.service.ts:72]: OK, can we go only with `endsWith('ibm.com')`?

Comment (pilartomas) [reply src/users/users.service.ts:72]: I don't think so, consider `foobar@foobarbaztestibm.com`

Comment (pilartomas) [reply src/users/users.service.ts:72]: what about `endsWith('.ibm.com')` ?

**Review by pilartomas — APPROVED** [review]

**Review by matoushavlena — APPROVED** [review]

**Pull request merged**